pkg/cmd/cli/install/install.go

```diff
@@ -347,5 +347,9 @@ func (o *InstallOptions) Validate(c *cobra.Command, args []string, f client.Fact
 		return errors.New("--default-restic-prune-frequency must be non-negative")
 	}
 
+	if len(o.Plugins) == 0 {
```

Comment:
I'm wondering if we should make this flag optional in the case where the user is not configuring a BSL or a VSL (i.e. L333 - https://github.com/vmware-tanzu/velero/pull/1985/files/f8b1e46469794dbc0250d4ddb9887d3760c7fcf2..006e1ca8b9b546e2ceeaeee8a352cdd351afcf00#diff-6dea8f0eef5112cb00bdce50e2cb339cR333).

Reply:
Yes, I guess so. If those are going to be set later, then we can let them add the plugins later too.

Reply:
Yeah, I think that makes sense. Let's make that change - then I think this is good to go!

Reply:
Done!